android/src/main/java/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -113,6 +118,25 @@ public Scheduler getScheduler() {
     return mScheduler;
   }
 
+  private void toggleSlowAnimations() {
+  }
+
+  private void addDevMenuOption() {
+    ((ReactApplication) mContext.get().getApplicationContext())
+        .getReactNativeHost()
+        .getReactInstanceManager()
+        .getDevSupportManager()
```

Comment:
plz introduce intermediate variables

Reply:
How do you want it to be split? One variable for devSupportManager, for each component or other way?

Reply:
I extracted devSupportManager as a variable in [be69639](https://github.com/software-mansion/react-native-reanimated/pull/2931/commits/be696393eb60a1f2cdeb4c4664fdcc8e1bf055c5).